src/SourceBuild/content/repo-projects/test-templates.proj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <LogVerbosityOptOut>true</LogVerbosityOptOut>
+
+    <BuildArgs Condition="'$(BuildOS)' == 'windows'">$(BuildArgs) $(FlagParameterPrefix)msbuildEngine dotnet</BuildArgs>
```

Comment:
Hmm, I'm confused. Isn't that the default?

Reply:
It's not. It is on non-Windows (option doesn't exist on non-windows), but on Windows build.ps1 will attempt to use desktop msbuild if there is a

```
"vs": {
      "version": "17.8",
}
```

element in the global.json

Reply:
We could make it the global default in the VMR build..I'll see about that.